src/lang/en-US.ts

```diff
@@ -930,6 +930,14 @@ const Workflows: ResolvedIntlConfig['messages'] = {
     'workflows.autoDiscovery.label.addNewBindings': `Automatically add new collections`,
     'workflows.autoDiscovery.label.evolveIncompatibleCollection': `Breaking changes re-version collections`,
 
+    'workflows.sourceCapture.header': `Source Capture`,
+    'workflows.sourceCapture.cta': `link capture`,
+    'workflows.sourceCapture.cta.edit': `update capture`,
+    'workflows.sourceCapture.selected.none': `no source capture`,
+    'workflows.sourceCapture.optin.message': `Below you can select a source capture to link to your materialization. This ensures as new collections are discovered on the capture, they will automatically be added to the materialization's bindings.`,
```

Comment:
Was this message requested by Dave or pulled from the documentation? I _believe_ there should be a comma between _ensures_ and _as_. This would apply to the similar piece of content below.

Reply:
Yup... it does.